lib/generators/blacklight/install_generator.rb

```diff
@@ -98,7 +98,8 @@ def inject_blacklight_i18n_strings
 
     def generate_blacklight_marc_demo
       if options[:marc]
-        gem "blacklight-marc", "~> 5.0"
+        blacklight_marc = String.new('blacklight-marc')
```

Comment:
Because rails mutates the string you pass in. And we have the frozen string pragma set.